{
  "uid" : "bf7ee10e924ca57b",
  "name" : "test_login01[case2]",
  "fullName" : "testcase.test_login.TestLogin#test_login01",
  "historyId" : "f1448aed770d7fdd91df4365d7b3461a",
  "time" : {
    "start" : 1650292731341,
    "stop" : 1650292731343,
    "duration" : 2
  },
  "status" : "broken",
  "statusMessage" : "NameError: name 'res_body' is not defined",
  "statusTrace" : "self = <testcase.test_login.TestLogin object at 0x000001AE41AFD280>\ncase = <comon.excel_util.CaseData object at 0x000001AE41A9B610>\n\n    @pytest.mark.parametrize('case',ExcelUtil().read_excel('login'))\n    def test_login01(self, case):\n        print(case)\n        allure.attach(body=case.url, name='接口路径')\n        allure.attach(body=case.case_data, name='请求参数')\n        allure.attach(body=case.res, name='预期结果')\n        logger.info(\"*************** 开始执行用例 ***************\")\n        # logger.info('----------->开始执行测试案例:{}'.format(case['case_title']))\n        response = requests.get(url=case.url, data=eval(case.case_data))\n        act_res=response.json()\n        print(act_res)\n>       allure.attach(body=str(res_body), name='响应结果')\nE       NameError: name 'res_body' is not defined\n\ntestcase\\test_login.py:27: NameError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "NameError: name 'res_body' is not defined",
    "statusTrace" : "self = <testcase.test_login.TestLogin object at 0x000001AE41AFD280>\ncase = <comon.excel_util.CaseData object at 0x000001AE41A9B610>\n\n    @pytest.mark.parametrize('case',ExcelUtil().read_excel('login'))\n    def test_login01(self, case):\n        print(case)\n        allure.attach(body=case.url, name='接口路径')\n        allure.attach(body=case.case_data, name='请求参数')\n        allure.attach(body=case.res, name='预期结果')\n        logger.info(\"*************** 开始执行用例 ***************\")\n        # logger.info('----------->开始执行测试案例:{}'.format(case['case_title']))\n        response = requests.get(url=case.url, data=eval(case.case_data))\n        act_res=response.json()\n        print(act_res)\n>       allure.attach(body=str(res_body), name='响应结果')\nE       NameError: name 'res_body' is not defined\n\ntestcase\\test_login.py:27: NameError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "f273bc008ec7a432",
      "name" : "接口路径",
      "source" : "f273bc008ec7a432.attach",
      "type" : "text/plain",
      "size" : 42
    }, {
      "uid" : "dba6fbe4160d1171",
      "name" : "请求参数",
      "source" : "dba6fbe4160d1171.attach",
      "type" : "text/plain",
      "size" : 167
    }, {
      "uid" : "551f5f224202d9c7",
      "name" : "预期结果",
      "source" : "551f5f224202d9c7.attach",
      "type" : "text/plain",
      "size" : 62
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "testcase"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "Mikey"
  }, {
    "name" : "thread",
    "value" : "9488-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcase.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "case",
    "value" : "<comon.excel_util.CaseData object at 0x000001AE41A9B610>"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "bf7ee10e924ca57b.json",
  "parameterValues" : [ "<comon.excel_util.CaseData object at 0x000001AE41A9B610>" ]
}